src/interpret.rs

```diff
@@ -6,19 +6,24 @@ use crate::command;
 use crate::errors;
 
 use failure::Error;
-
+use std::path;
 
 /// Build a spec for the docker container, and then run it
-pub(crate) fn run_container(config: &FlokiConfig, environ: &Environment, command: &str) -> Result<(), Error> {
-    let (mut cmd, mut dind) = build_basic_command(&config, &environ);
+pub(crate) fn run_container(
+    environ: &Environment,
+    floki_root: &path::Path,
+    config: &FlokiConfig,
+    command: &str,
+) -> Result<(), Error> {
+    let (mut cmd, mut dind) = build_basic_command(&floki_root, &config);
 
     cmd = configure_dind(cmd, &config, &mut dind)?;
     cmd = configure_floki_user_env(cmd, &environ);
-    cmd = configure_floki_workdir_env(cmd, &environ);
+    cmd = configure_floki_workdir_env(cmd, &floki_root);
```

Comment:
It would be nearer to avoid passing it around separately. However, `floki_root` depends on the location of `floki.yaml` and so the `-c` argument, which seemed a bit out of the purview of `Environment`. Let me know if you disagree

Reply:
Yeah, I think it's fine for now. Easy enough to move later if wanted.